### PR TITLE
Pull Request for Multiple Result Submission First Phase

### PR DESCRIPTION
**Pull request by Abidemi**
Hi Rapo,

I am making this pull request to let you know that I have completed the work on Multiple Result Submission and have pushed all files to this repository.

Many thanks

Abi

### CODE REVIEW

**Comment by kaselis** [comment]
Hi Abi, could you make sure that all tests pass as well? Thanks.

**Comment by Abidemi** [comment]
Hi Rapo,

All checks have now passed.

Kind regards.

Abi

**Comment by kaselis** [comment]
It doesn't look like it is sorting (or rather leaving sort out). Response from the server comes sorted by weight, but then it is sorted by name on the frontend.

**Comment by Abidemi** [comment]
Hi Rapo,
Sorting by weight at the frontend is now fixed.

File: src/app/patients/results/observation-selector.html [hunk 1]
```diff
@@ -1,73 +1,163 @@
 
-
+<link href="Assets/StyleSheets/observation-selector.css" rel="stylesheet" />
```
Comment (kaselis) [inline src/app/patients/results/observation-selector.html]: This is not a valid location for a link tag. Also styles should go into src/sass folder, not to some random place. If you want to leave them here, you'll have to modify the build process to include in the output, though I do not recommend this way.

File: src/app/patients/results/observation-selector.directive.js [hunk 2]
```diff
@@ -381,7 +381,7 @@ function observationSelector(store) {
           });
 
           // Get's patients groups sorted by name
-          scope.groups = _.sortBy(scope.patient.getGroups(), 'shortName');
+          scope.groups = _.sortBy(scope.patient.getGroups(), 'weight');
```
Comment (kaselis) [inline src/app/patients/results/observation-selector.directive.js]: This change sorts all observations by weight (if weight is provided), including in the all tab. However, in the all tab, observations should be sorted by name alphabetically by default.

**Comment by Abidemi** [comment]
Hi Rapo,

This is a pull request to let you know that that I have effected changes for items to be displayed from top to bottom.

Kind regards.

Abi

**Comment by kaselis** [comment]
Could you, please, make sure that code style is consistent with the rest of the repo. Right now it is not, we use 2 space indentation for html and js files, js file inconsistently put brackets on the same line and new line, etc.

**Comment by Abidemi** [comment]
Hi Rapo,

This is to let you know that I have re-formatted html and js files, they are now consistent with other files in the repo. kind regards.

Abi

**Comment by kaselis** [comment]
Hmm, ordering still doesn't work. It seems to take the smallest weight of all groups, no matter which group data I'm looking at. As an example can be seen in Nurture CKD cohort, and LDL Cholesterol, has a weight of 62, yet it appears second in the list. I suspect this is because of weight 10 in Autosomal Dominant Tubulointerstitial Kidney Disease (FUAN) group. If I change ADTKD group weight to something big, like 100, then that observation (LDL Cholesterol) correctly appears in Nurture CKD list, between 61 and 63 weights.

**Comment by Abidemi** [comment]
Update for Radar Observation weight.

**Comment by Abidemi** [comment]
Update for Radar Observation weight.

**Comment by kaselis** [comment]
Doesn't look like those changes made any difference. Also, could you merge master into your branch and resolve conflicts.

**Comment by Abidemi** [comment]
Hi Rapo,
This is to let you know that I have merge main-master with all the updates I added from May. 

Unfortunately, the issue with weight sorting has not been resolved.

Kind regards.

Abi

**Comment by kaselis** [comment]
Not sure how you have merged master, but conflicts are still there, different, but still present.

**Comment by Abidemi** [comment]
I downloaded the latest (updated on 19 Sept 2017 by you). I applied all the changes I have made since May  2017 to date, uploaded it to my forked  branch and the made a pull request.

**Pull request closed without merging**